<launch>
    <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="gui" value="false" />
        <arg name="world_name" value="$(find collect_data)/launch/dynamic_multiview_3d.world" />
        <!--<arg name="throttled" value="true" />-->
    </include>
    <node pkg="collect_data" type="manage_objects.py" name="manage_objects" output="screen" />
</launch>
